app/src/main/java/com/monkeyteam/chimpagne/ui/components/Button.kt

```diff
@@ -106,6 +106,9 @@ fun IconTextButton(
       }
 }
 
+@Deprecated(
```

Comment:
Nice documentation, it will be useful when refactoring to follow the new paradigm